anti_patterns/deny-warnings.md

```diff
@@ -35,7 +35,10 @@ before there was none.
 All this conspires to potentially break the build whenever something changes.
 
 Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
-longer be used unless the annotation is removed.
+longer be used directly. Note that there is a way around this: With the
+`--cap-lints=warn` command line argument, all `deny` lint errors are turned into
+warnings (there may still be `forbid` lints that will stop compilation, but
+those are a different topic).
```

Comment:
Either you should explain the difference between deny and forbid, or not mention forbid at all